src/framework/tooltip/Tooltip.jsx

```diff
@@ -132,11 +149,15 @@ export default class Tooltip extends Component {
     return (
       <div
         className="tooltip"
-        onMouseEnter={this.onMouseEnter}
-        onMouseLeave={this.onMouseLeave}
         ref={(div) => { this.tooltip = div; }}
       >
-        {this.props.children}
+        <span
+          onMouseEnter={this.onMouseEnter}
+          onMouseLeave={this.onMouseLeave}
+        >
```

Comment:
As discussed the mouse events are only related to the `children` and not `container` (had some side effects on scrolling).